assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -388,6 +388,35 @@ const baseSelectors = {
 		return modules[ slug ];
 	} ),
 
+	/**
+	 * Gets a specific module icon by slug.
+	 *
+	 * Returns a specific module icon by its slug.
+	 * Returns `null` if state is still loading or if said module doesn't exist or doesn't have an icon.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(WPComponent|null|undefined)} A specific module's icon; `undefined` if state is still loading or if said module doesn't exist; `null` if it doesn't have an icon.
```

Comment:
The explanation here is not exactly correct based on the code below: `undefined` is only returned if state is still loading. `null` is returned if the module doesn't exist or if it doesn't have an icon.